projects/storefrontapp/src/app/spartacus/features/cart-feature.module.ts

```diff
@@ -24,6 +32,22 @@ import { I18nConfig, provideConfig } from '@spartacus/core';
         fallbackLang: 'en',
       },
     }),
+    provideConfig({
```

Comment:
Should wiish list get it's own `wishlist-feature.module.ts` file like other features like saved cart?